### PR TITLE
fix Scryfall scraper

### PR DESCRIPTION
**Pull request by dev-id**
fixing https://github.com/Cockatrice/Magic-Spoiler/issues/164

### CODE REVIEW

File: scryfall_scraper.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import time
 
 
-def get_scryfall(setUrl):
+def get_scryfall(setUrl='https://api.scryfall.com/cards/search?q=++e:xln'):
     #getUrl = 'https://api.scryfall.com/cards/search?q=++e:'
     #setUrl = getUrl + code.lower()
```
Comment (tooomm) [inline scryfall_scraper.py:8]: We don't want to hard code a set here.
The commented out code looks fine and should do the trick, no?

File: scryfall_scraper.py [hunk 2]
```diff
@@ -16,30 +16,56 @@ def get_scryfall(setUrl):
             scryfall.append(setcards['data'])
         else:
             setDone = True
-            # print setUrl
-            # print setcards
```
Comment (tooomm) [inline scryfall_scraper.py:20]: Printing the api url where we get the data from doesn't look too bad :)

`print setcards` would print all card names though?!

**Review by tooomm — DISMISSED** [review]

**Comment by tooomm** [comment]
Let's decouple things here first!

`scryfallOnly` switchs can be easily flipped later.

File: scryfall_scraper.py (hunk 1 above)
Comment (dev-id) [reply scryfall_scraper.py:8]: This is just a default setting to help with debug

Actual URL is set here: https://github.com/Cockatrice/Magic-Spoiler/blob/ed2ea3945a194333b79ab593c62ae384d1a1412c/main.py#L134

File: scryfall_scraper.py (hunk 2 above)
Comment (dev-id) [reply scryfall_scraper.py:20]: If everything's working, it's excessive debug printing

File: scryfall_scraper.py (hunk 1 above)
Comment (tooomm) [reply scryfall_scraper.py:8]: Didn't check the not changed files 👍

**Comment by tooomm** [comment]
What the heck... where does this manual added cards come from!?
https://travis-ci.org/Cockatrice/Magic-Spoiler/builds/276518359#L485-L489

There should be no single xln card at all, I turned the set off. 
Since the manual adds are the only cards which are added for xln, turning off the scrapers worked.

But I think I spot a new issue here! :)
Our `cards_manual` file is empty in this branch (https://github.com/Cockatrice/Magic-Spoiler/blob/tooomm-scryfallOnly/cards_manual.yml) but we still get manual cards added.
So I guess the code always pulls this file from master, instead the correct branch the rest is running from.

**Comment by dev-id** [comment]
This commit was using this manual file: https://github.com/Cockatrice/Magic-Spoiler/blob/7f1f81ff23223bed58beb136df3e31ac2f4b6e70/cards_manual.yml

**Review by dev-id — COMMENTED** [review]
Only a default value, makes the function better-handled for testing/debug.

**Comment by tooomm** [comment]
>This commit was using this manual file: /cards_manual.yml@7f1f81f

I can't localize or match that file.
But why isn't it using the cards_manual.yml file from the same branch we are running the code from!?

Do I miss something useful here why we wouldn't want to do that?

**Comment by dev-id** [comment]
It is using the branch and commit file, if you visit that travis build (https://travis-ci.org/Cockatrice/Magic-Spoiler/builds/276518359#L485-L489) and click on the commit (https://github.com/Cockatrice/Magic-Spoiler/commit/4c149b9952f4f2575cecfcdc67506568722d05c4) the view the cards_manual for that commit, it has manual cards in it.

**Comment by tooomm** [comment]
So i guess we are ok here.

**Comment by tooomm** [comment]
now there are no manual cards added any more... !?
https://travis-ci.org/Cockatrice/Magic-Spoiler/builds/276550512#L488-L492

**Comment by dev-id** [comment]
Correct

https://github.com/Cockatrice/Magic-Spoiler/blob/cc3cd6818c3373f25f47fa340cd536666076e350/cards_manual.yml

https://github.com/Cockatrice/Magic-Spoiler/blob/tooomm-scryfallOnly/cards_manual.yml

**Pull request merged**